epitator/utils.py

```diff
@@ -56,7 +56,7 @@ def parse_number(num, default=None):
 def parse_spelled_number(num_str):
     """Parse spelled out whole numbers."""
     tokens = []
-    for t in num_str.split(' '):
+    for t in num_str.strip().split(' '):
```

Comment:
I prefer the explicit argument since it makes it clear what the command does without the need for documentation. In other languages like JS split with no argument has a different behavior.